crates/sui-storage/src/indexes.rs

```diff
@@ -845,4 +845,11 @@ impl IndexStore {
     pub fn is_empty(&self) -> bool {
         self.tables.owner_index.is_empty()
     }
+
+    pub fn checkpoint_db(&self, path: &Path) -> SuiResult {
+        self.tables
+            .transactions_from_addr
```

Comment:
only `transactions_from_addr`?

Reply:
No it just uses the column family name to get the underlying db, it is checkpointing every table in db indeed

Reply:
it see. Would be great to add a comment for future readers